lib/v2/dockerhub/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'hub.docker.com': {
+        _name: 'Docker Hub',
+        '.': [
+            {
+                title: '镜像有新 Tag',
+                docs: 'https://docs.rsshub.app/program-update.html#docker-hub-jing-xiang-you-xin-tag',
```

Comment:
```suggestion
                docs: 'https://docs.rsshub.app/program-update.html#docker-hub',
```
This should point to H2 of the docs 
See https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze-bian-xie-gui-ze-docs